libvmi/libvmi.h

```diff
@@ -481,6 +481,7 @@ typedef uint64_t reg_t;
 #define SPSR_EL1    SPSR_SVC
 #define TCR_EL1     TTBCR
 
+typedef struct hvm_hw_mtrr mtrr_regs_t;
```

Comment:
This is no good. Are you pulling this in from a Xen header? What about non-Xen systems?

Reply:
I've amended the pull request with a custom libvmi mtrr regs structure and moved the xen specific structure use in the xen driver.